bin/cml-send-comment.js

```diff
@@ -36,10 +41,10 @@ const opts = yargs
   .default('token')
   .describe(
     'token',
-    'Personal access token to be used. If not specified in extracted from ENV REPO_TOKEN.'
+    'Personal access token to be used. If not specified is extracted from ENV REPO_TOKEN.'
   )
   .default('driver')
-  .choices('driver', ['github', 'gitlab'])
+  .choices('driver', ['github', 'gitlab', 'bitbucket'])
```

Comment:
Bitbucket support wasn't enabled at the command level. Was it intentional? 🤔

Reply:
thats very weird! Indeed bitbucket should not work and it does!
strict seems not be be applying here?

Reply:
> _Indeed Bitbucket should not work and it does!_

It does work, at least for `cml-send-comment` 🙃  I thoroughly tested it yesterday and it also passes the automated checks.

> _strict seems not be be applying here?_

What do you mean?

Reply:
yargs has strict, if we were setting a weird value it should not like it. Thats the goal of choices here

Reply:
And it works as intended. In fact, tests don't invoke the command-lin tool but the `cml` module. That's why it doesn't trip yargs when running Bitbucket tests.